configs/bundle_config.yaml

```diff
@@ -283,3 +283,41 @@ databundles:
     output: [cutouts/cutout-2013-era5.nc]
     disable_by_opt:
       build_cutout: [all]
+
+  # # cutouts bundle of the cutouts folder for the African continent
+  # # This bundle includes SC, ST, KM and CV countries into the previous bundle
```

Comment:
I fear it may be not completely clear what is meant as `previous bundle`. Would it make sense to clarify that a "basic" continental version is meant?

Reply:
Done :)